examples/serve/spot-dynamic-failover-on-demand-fallback.yaml

```diff
@@ -0,0 +1,22 @@
+service:
+  readiness_probe:
+    path: /health
+    initial_delay_seconds: 20
+  replica_policy:
+    min_replicas: 1
```

Comment:
In the simulator I keep `min_replicas: 3`, `max_replicas:6`. I think for end-to-end it's probably fine to do it from 1 replica to 3 replicas.